src/app-logic/l10n-bundling.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import engFtlFile from '../../locales/en-US/app.ftl';
+
+export async function fetchMessages(locale: string): Promise<[string, string]> {
+  const response = await fetch(engFtlFile);
+  const messages = await response.text();
+  return [locale, messages];
+}
```

Comment:
Currently this function gets only the English ftl file and not others. If you add more ftl files, they will not work because of this. We should make this more generic for other locales.
I guess we can keep it for now and do this iteratively. But we should at least add a FIXME comment here to mention this.